backend/src/main/java/ca/mcgill/purposeful/configuration/SecurityConfiguration.java

```diff
@@ -115,6 +116,24 @@ JwtDecoder jwtDecoder() throws Exception {
     return NimbusJwtDecoder.withPublicKey(rsaKeys().getPublicKey()).build();
   }
 
+  /**
+   * Allows localhost:3000 to access the backend,
+   * Allows GET and POST requests, add others if necessary
+   * Allows Authorization header (Basic, Bearer)
+   * @return
+   */
+  @Bean
+  CorsConfigurationSource corsConfigurationSource() {
+    CorsConfiguration configuration = new CorsConfiguration();
+    configuration.setAllowedOrigins(Arrays.asList("http://localhost:3000/"));
+    configuration.setAllowedMethods(Arrays.asList("GET","POST"));
```

Comment:
Do we also need to allow put requests and delete requests? I believe we have them both

Reply:
Yes, done